frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -275,22 +291,24 @@ export default class ProblemDetails extends Vue {
   @Prop({ default: 0 }) availableTokens!: number;
   @Prop({ default: 0 }) allTokens!: number;
   @Prop() histogram!: types.Histogram;
-  @Prop() showNewRunWindow!: boolean;
+  @Prop({ default: PopupDisplayed.None })
+  initialPopupDisplayed!: PopupDisplayed;
   @Prop() activeTab!: string;
   @Prop() allowUserAddTags!: boolean;
   @Prop() levelTags!: string[];
   @Prop() problemLevel!: string;
   @Prop() publicTags!: string[];
   @Prop() selectedPublicTags!: string[];
   @Prop() selectedPrivateTags!: string[];
+  @Prop() localNominated!: boolean;
```

Comment:
cuál es la razón del prefijo "local"? existe el concepto de nominación no-local o remota? _si entiendo bien_ lo que está haciendo el `.ts`, esto es más bien si el usuario ya hizo la nominación para saber si mostrar el enlace. si ese es el propósito, creo que sería mejor

```suggestion
  @Prop() hasBeenNominated!: boolean;
```

en general cuando se use un calificador (o adjetivo) para describir una variable, hay que pensar si el opuesto / una alternativa de ese calificador (o la variable sin calificador) tiene sentido. si no, es mejor buscar otro o eliminarlo.

Reply:
Es correcto, se utiliza para saber si el usuario ya lo nominó, así que queda mejor ese nombre.